tests/functional_tests/helper_scripts/website_load/status_check.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
As per suggestion given in one of the previous comments, please move this to a separate PR.